### PR TITLE
Helm chart added

### PR DESCRIPTION
**Pull request by thundersparkf**
Basic Helm chart added. Tested on locally setup Minikube cluster and Persistent Volumes for data storage.
Signed-off-by: thundersparkf <devagastya0@gmail.com>

Fixes https://github.com/chatwoot/chatwoot/issues/642

### CODE REVIEW

**Comment by thundersparkf** [comment]
@pranavrajs please let me know if there's any issues. I'm new to Helm too so if there's any convention for naming the deployments of services I'd be happy to learn that and use it from here on.

**Comment by pranavrajs** [comment]
@thundersparkf Thanks for the work! This is great. Let me try to go over the PR this week. I'll let you know if I find anything :)

**Comment by thundersparkf** [comment]
Fixed Postgres and Redis deployments being triggered even if storage is not local.

**Comment by akinomeroglu** [comment]
Hi all,

Do you need any help / test for this issue?

**Comment by sojan-official** [comment]
@akinomeroglu  That would be super helpful.

**Comment by akinomeroglu** [comment]
Hi @sojan-official 

Sorry for my late response. We will deploy chatwoot with this chart and come back to you with our recommendation and notes in the first half of this week.

**Comment by thundersparkf** [comment]
@akinomeroglu how's it going? Any issues?

**Comment by bilalunalnet** [comment]
Hi,

We managed to deploy chatwoot with helm chart. There were some minor problems that i noted;

- I had to edit the sizes of the persistent volume claims in order to use DigitalOcean volumes. Apparently there is a minimum limit of  volume request sizes. This value is defined in values.yml and can be changed easily, but it is better to increase the default value to 1Gi maybe?
- I had to add new environment variable named "DATABASE_URL" because I couldn't find where to define the database port.
- I had to remove initContainers for sidekiq and rails deployments because they were not running as intended. `nslookup` command trying to reach hardcoded postgres and redis hosts which I didn't deployed.

I deployed services without any problems, except for the above items. Thanks @thundersparkf for your effort.

**Comment by thundersparkf** [comment]
@bilalunalnet Thanks for the review! I'll fix these soon!

**Comment by akinomeroglu** [comment]
Hi @thundersparkf,

Additionally One minor advice, maybe you cat set the frontend url (FRONTEND_URL=http://0.0.0.0:3000 by default) in values file. 

Ps. we are not experienced chatwoot users atm so there could be some glitches but it seems that service is up and running right now.

**Comment by thundersparkf** [comment]
Hi @akinomeroglu !

Yes the FRONTEND_URL is already set to that value. 

Screenshot 2021-06-29 at 7 30 14 PM

Is this what you mean?

**Comment by akinomeroglu** [comment]
> Hi @akinomeroglu !
> 
> Yes the FRONTEND_URL is already set to that value.
> 
> Screenshot 2021-06-29 at 7 30 14 PM
> 
> Is this what you mean?

Yes, I missed that sorry for trouble.

**Comment by thundersparkf** [comment]
@bilalunalnet Can you elaborate on this issue? I'm not sure I get it

> * I had to add new environment variable named "DATABASE_URL" because I couldn't find where to define the database port.

**Comment by vishnu-narayanan** [comment]
@thundersparkf could you please remove the hardcoded Redis hostname ?

File: values.yaml [hunk 1]
```diff
@@ -183,7 +183,7 @@ env:
   REDIS_SENTINEL_MASTER_NAME: ""
   REDIS_SENTINELS: ""
   REDIS_HOST: 15.206.207.127
```
Comment (vishnu-narayanan) [inline values.yaml:185]: could you please remove the hardcode redis host name ?

Comment (thundersparkf) [reply values.yaml]: Done!

**Comment by vishnu-narayanan** [comment]
@thundersparkf Thank you for the changes. There are some minor issues with running migration against `postgres` when deploying in a managed cluster. Lets us merge this to develop and take it from there.
cc: @sojan-official

**Review by vishnu-narayanan — APPROVED** [review]

**Review by sojan-official — APPROVED** [review]

**Pull request merged**